oracle/config/market.go

```diff
@@ -0,0 +1,102 @@
+package config
+
+import (
+	"fmt"
+
+	oracletypes "github.com/skip-mev/slinky/x/oracle/types"
+)
+
+// MarketConfig represents the provider specific configurations for different
+// currency pairs and the corresponding markets they are traded on.
+type MarketConfig struct {
+	// Name identifies which provider this config is for.
+	Name string `mapstructure:"name" toml:"name"`
+
+	// CurrencyPairToMarketConfigs is the config the provider uses to create mappings
+	// between on-chain and off-chain currency pairs. In particular, this config
+	// maps the on-chain currency pair representation (i.e. BITCOIN/USD) to the
+	// off-chain currency pair representation (i.e. BTC/USD).
+	CurrencyPairToMarketConfigs map[string]CurrencyPairMarketConfig `mapstructure:"currency_pair_to_market_configs" toml:"currency_pair_to_market_configs"`
+}
+
+// CurrencyPairMarketConfig is the config the provider uses to create mappings
+// between on-chain and off-chain currency pairs.
+type CurrencyPairMarketConfig struct {
+	// Ticker is the ticker symbol for the currency pair off-chain.
+	Ticker string `mapstructure:"ticker" toml:"ticker"`
+
+	// CurrencyPair is the on-chain representation of the currency pair.
+	CurrencyPair oracletypes.CurrencyPair `mapstructure:"currency_pair" toml:"currency_pair"`
+}
```

Comment:
We can later extend this to do something like provider specific cross market aggregation, or inversion by market (i.e. we want ETH/BTC but only BTC/ETH is available).